tests/cephfs/cephfs_top/cephfs_top_initiate.py

```diff
@@ -0,0 +1,142 @@
+import random
+import string
+import traceback
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from utility.log import Log
+
+log = Log(__name__)
+
+"""
+Testing description:
+
+Verify cephfs-top initial status and basic configuration
+
+Steps to Reproduce:
+1. Try to test install cephfs-top package
+2. Cehck if cephfs-top packages install is working well
+3. Check if cephfs-top is working without enabling mgr stats
+4. check if cephfs-top client is created well with appropriate permission
+5. check if cephfs-top -h,--help is working well
```

Comment:
There are some more steps in https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83573829 that needs to be considered in the automation as below,

run "ceph auth ls" \| grep fstop | fstop client should be created after the installation
run "cephfs-top" in the client node | cephfs-top should be working
Create a client for non-default user | creating a client should be successful
run "cephfs-top --id <client>" | operation should be successful
Check if cephfs-top manual is present with detailed summary | all the summary should be listed with clear descriptions
Uninstall cephfs-top | Uninstall should be sucessful
run "cephfs-top" | Should not be working

Are these being covered in other automation?